### PR TITLE
update some sources

### PR DESCRIPTION
**Pull request by delphiactual**
update some sources due to confusion on vendor screen

### CODE REVIEW

File: data/sources/unassigned.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
   "164083100": "Source: Display of Supremacy, Weekly Challenge",
```
Comment (delphiactual) [inline data/sources/unassigned.json:2]: not sure what this is yet.

File: data/sources/unassigned.json [hunk 2]
```diff
@@ -1,6 +1,5 @@
...
-  "2939318890": "Source: Wayfinder's Compass",
   "3094114967": "Source: Season of the Lost ritual playlists",
   "3100439379": "Source: Mission Exorcism",
```
Comment (delphiactual) [inline data/sources/unassigned.json:4]: not sure what this is yet.

File: data/sources/unassigned.json [hunk 3]
```diff
@@ -1,6 +1,5 @@
 {
   "164083100": "Source: Display of Supremacy, Weekly Challenge",
-  "2939318890": "Source: Wayfinder's Compass",
   "3094114967": "Source: Season of the Lost ritual playlists",
```
Comment (delphiactual) [inline data/sources/unassigned.json:3]: is this just shattered realm and astral alignment?

File: data/sources/unassigned.json [hunk 4]
```diff
@@ -1,6 +1,5 @@
...
   "3100439379": "Source: Mission Exorcism",
   "3845969330": "Source: Xûr"
```
Comment (delphiactual) [inline data/sources/unassigned.json:5]: should we add this?

File: data/sources/unassigned.json (hunk 1 above)
Comment (nev-r) [reply data/sources/unassigned.json:2]: maybe vanguard ritual weapon ornament, maybe the seasonal exotic quest from mara, a quick manifest dive on this is confusing

File: data/sources/unassigned.json (hunk 2 above)
Comment (nev-r) [reply data/sources/unassigned.json:4]: if you mean the Exorcism line, also confusing

Comment (delphiactual) [reply data/sources/unassigned.json:4]: yes

File: data/sources/unassigned.json (hunk 3 above)
Comment (nev-r) [reply data/sources/unassigned.json:3]: 3094114967? i dont know this is weird. the terminology has gotten strange over there..... it's definitely the set of seasonal gear but not sure exactly what they are trying to imply with the source string

File: data/sources/unassigned.json (hunk 4 above)
Comment (nev-r) [reply data/sources/unassigned.json:5]: that's weird really weird. i feel strange about what items are marked as 3845969330

**Pull request merged**